Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM debian:bookworm-slim AS base
+FROM debian:latest AS base
```

Comment:
Any reason for removing slim? (latest is currently bookworm)

Reply:
@knorrium

Reply:
@junderw I misread your original message saying it was due to the ancient version Debian was using, so I ended up changing the Debian version without checking if bookworm wasn't already the latest. I'll change it back to slim.